answerking/urls.py

```diff
@@ -1,23 +1,23 @@
-"""answerking URL Configuration
+from django.contrib import admin
+from django.urls import include, path, re_path
 
-The `urlpatterns` list routes URLs to views. For more information please see:
-    https://docs.djangoproject.com/en/4.0/topics/http/urls/
-Examples:
-Function views
-    1. Add an import:  from my_app import views
-    2. Add a URL to urlpatterns:  path('', views.home, name='home')
-Class-based views
-    1. Add an import:  from other_app.views import Home
-    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
-Including another URLconf
-    1. Import the include() function: from django.urls import include, path
-    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
-"""
+from drf_yasg import openapi
+from drf_yasg.views import get_schema_view
+from rest_framework import permissions
 
-from django.contrib import admin
-from django.urls import include, path
+schema_view = get_schema_view(
+   openapi.Info(
+      title="AnswerKing Python API",
+      default_version='v1',
+      description="Test description",
+   ),
+   public=True,
+   permission_classes=[permissions.AllowAny],
+)
```

Comment:
what do `public=True` and `permission_classes=[permissions.AllowAny]` do?

Reply:
Adds permissions to view the schema and the swagger endpoints, they were included in the installation directions in the documentation so included them. Not entirely sure what happens if public is set to False. 
https://drf-yasg.readthedocs.io/en/stable/readme.html#installation

Reply:
I think they allow swagger to be fully accessed hence including the DB, it has to be eventually disabled during deployment

Reply:
right now the dependency is in development but the import is actually generic it would throw an exceptions there. I workaroud is doing those things in a `if DEBUG:` block I guess but it's a bit ugly

Reply:
```
if DEBUG:
   from drf_yasg import openapi
   from drf_yasg.views import get_schema_view
   from rest_framework import permissions
   schema_view = get_schema_view(
   openapi.Info(
      title="AnswerKing Python API",
      default_version='v1',
      description="Test description",
   ),
   public=True,
   permission_classes=[permissions.AllowAny],)
   urlpatterns += [re_path(r'^swagger(?P<format>\.json|\.yaml)$', schema_view.without_ui(cache_timeout=0), name='schema-json'),
    path(r'api/swagger', schema_view.with_ui('swagger', cache_timeout=0), name='schema-swagger-ui')]
```

But it's super ugly perhaps the library supports deployment mode by itself

Reply:
That is a good point, we will need to do this for coverage too as it's a dev dependency but included in installed apps. Going to look into splitting our settings.py into 2 files, one for dev and one for production as part of this ticket

Reply:
don't need to split in a separate file tho, you can add a DEBUG block in the settings.py too

Reply:
A standard way doesn't seem to exist https://stackoverflow.com/questions/1626326/how-to-manage-local-vs-production-settings-in-django?rq=1 I guess your solution would be alright then